### PR TITLE
MKM 20 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Behind the Mask
+ManaCost:U
+Types:Instant
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. (Exile cards with total mana value 6 or greater from your graveyard.)
+A:SP$ Animate | ValidTgts$ Artifact,Creature | TgtPrompt$ Select target artifact or creature | Power$ 4 | Toughness$ 3 | ConditionDefined$ Collected | ConditionPresent$ Card | Types$ Artifact,Creature | SubAbility$ DBAnimateBis | SpellDescription$ Until end of turn, target artifact or creature becomes an artifact creature with base power and toughness 4/3.
+SVar:DBAnimateBis:DB$ Animate | Power$ 1 | Toughness$ 1 | Defined$ ParentTarget | ConditionDefined$ Collected | ConditionPresent$ Card | ConditionCompare$ EQ0 | Types$ Artifact,Creature | SpellDescription$ If evidence was collected, it has base power and toughness 1/1 until end of turn instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: wrong Compare

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: should just use a single effect

File: forge-gui/res/cardsfolder/upcoming/leering_onlooker.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Leering Onlooker
+ManaCost:1 B
+Types:Creature Vampire
+PT:1/3
+K:Flying
+A:AB$ Token | Cost$ 2 B B ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | TokenAmount$ 2 | TokenScript$ b_1_1_bat_flying | SpellDescription$ Create two tapped 1/1 black Bat creature tokens with flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/leering_onlooker.txt:6]: tapped missing

File: forge-gui/res/cardsfolder/upcoming/yarus_roar_of_the_old_gods.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.faceDown+YouCtrl | ValidTarget$ Player | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever one or more face-down creatures you control deal combat damage to a player, draw a card.
+SVar:TrigDraw:DB$ Draw | NumCards$ 1
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.faceDown+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigReturn | TriggerDescription$ Whenever a face-down creature you control dies, return it to the battlefield face down under its owner's control if it's a permanent card, then turn it face up.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | RememberChanged$ True | Defined$ TriggeredCard | SubAbility$ DBTurnFaceUp | ConditionDefined$ TriggeredCard | ConditionPresent$ Card.Permanent
+SVar:DBTurnFaceUp:DB$ SetState | Defined$ Remembered | Mode$ TurnFaceUp | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHints:Keyword$Morph|Disguise & Ability$Morph
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yarus_roar_of_the_old_gods.txt:12]: Ability$Morph doesn't exist

File: forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: Yeah with Power$ X and Toughness Y but coudn't figure out how to make that work

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: should work normally now

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: any pointers ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/behind_the_mask.txt:6]: `Collected$Amount`

File: forge-gui/res/cardsfolder/upcoming/barbed_servitor.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Name:Barbed Servitor
+ManaCost:3 B
+Types:Artifact Creature Construct
+PT:1/1
+K:Indestructible
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSuspect | TriggerDescription$ When CARDNAME enters the battlefield, suspect it. (It has menace and can't block.)
+SVar:TrigSuspect:DB$ AlterAttribute | Defined$ Self | Attributes$ Suspected
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you draw a card and you lose 1 life.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBLoseLife | SpellDescription$ You draw a card and you lose 1 life.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/barbed_servitor.txt:9]: spelldescription on trigger

File: forge-gui/res/cardsfolder/upcoming/basilica_stalker.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Basilica Stalker
+ManaCost:5 B
+Types:Creature Vampire Detective
+PT:3/4
+K:Flying
+K:Disguise:4 B
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Battle | CombatDamage$ True | Execute$ TrigGainLife | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you gain 1 life and surveil 1. (Look at the top card of your library. You may put it into your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/basilica_stalker.txt:7]: wrong target

File: forge-gui/res/cardsfolder/upcoming/branch_of_vitu_ghazi.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Branch of Vitu-Ghazi
+ManaCost:no cost
+Types:Land
+K:Disguise:3
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+T:Mode$ TurnFaceUp | ValidCard$ Card.Self | Execute$ TrigMana | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME is turned face up, add two mana of any one color. Until end of turn, you don't lose this mana as steps and phases end.
+SVar:TrigMana:DB$ Mana | Produced$ Combo Any | Amount$ 2 | PersistentMana$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/branch_of_vitu_ghazi.txt:7]: wrong produced

File: forge-gui/res/cardsfolder/upcoming/yarus_roar_of_the_old_gods.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/yarus_roar_of_the_old_gods.txt]: come on...Ability$Manifest doesn't exist either

**Review by tool4ever — APPROVED** [review]

**Pull request merged**